helm/test-values.yaml

```diff
@@ -0,0 +1,249 @@
+    REACT_APP_CTFD_URL: "https://ctfd.io" #requierd for 2 and 3 domain setup
+    REACT_APP_S3_BUCKET_URL: "s3://funstuff" #the s3 bucket you use for teh aws challenges, don't forget to make it accessible!
+    K8S_ENV: "k8s" #or 'aws'
+    REACT_APP_ACCESS_PASSWORD: "" #DEFAULT NO PASSWORD, PLAYING THIS IN PUBLIC? PUT A FANCY STRING HERE, BUT BE GENTLE: USERS NEED TO BE ABLE TO COPY THAT STUFF...
+    REACT_APP_CREATE_TEAM_HMAC_KEY: "hardcodedkey"
+    IRSA_ROLE: arn:aws:iam::233483431651:role/wrongsecrets-secret-manager #change this in your own AWS role!
+    SECRETS_MANAGER_SECRET_ID_1: "wrongsecret" #only change if you need non-default AWS SM entries
+    SECRETS_MANAGER_SECRET_ID_2: "wrongsecret-2" #only change if you need non-default AWS SM entries
+  metrics:
+    # -- enables prometheus metrics for the balancer. If set to true you should change the prometheus-scraper password
+    enabled: true
+    dashboards:
+      # -- if true, creates a Grafana Dashboard Config Map. (also requires metrics.enabled to be true). These will automatically be imported by Grafana when using the Grafana helm chart, see: https://github.com/helm/charts/tree/main/stable/grafana#sidecar-for-dashboards
+      enabled: false
+    serviceMonitor:
+      # -- If true, creates a Prometheus Operator ServiceMonitor (also requires metrics.enabled to be true). This will also deploy a servicemonitor which monitors metrics from the Juice Shop instances
+      enabled: false
+    basicAuth:
+      username: prometheus-scraper
+      # -- Should be changed when metrics are enabled.
+      password: ERzCT4pwBDxfCKRGmfrMa8KQ8sXf8GKy
+
+##TODO:
+#1. UPDATE WRONGSECRETS TO DEFINITION WHERE K8S IS USED FOR AWS!
+#2. UPDATE VIRTUALDESKTOP TO DEFINITION WHERE IT CN BE USED
+#3. ADD VAULT INSTANCE PER TEAM!
+
+wrongsecrets:
+  # -- Specifies how many Wrongsecrets instances should start at max. Set to -1 to remove the max Wrongsecrets instance cap
+  maxInstances: 500
+  # -- Wrongsecrets Image to use
+  image: jeroenwillemsen/wrongsecrets
+  tag: 1.6.4-no-vault
+  # -- Change the key when hosting a CTF event. This key gets used to generate the challenge flags. See: https://github.com/OWASP/wrongsecrets#ctf
+  ctfKey: "zLp@.-6fMW6L-7R3b!9uR_K!NfkkTr"
+  # -- Specify a custom Juice Shop config.yaml. See the JuiceShop Config Docs for more detail: https://pwning.owasp-juice.shop/part1/customization.html#yaml-configuration-file
+  # @default -- See values.yaml for full details
+  config: |
+    K8S_ENV: aws
+  #    application:
+  #      logo: https://raw.githubusercontent.com/iteratec/multi-juicer/main/images/multijuicer-icon-only-padding.png
+  #      favicon: https://raw.githubusercontent.com/iteratec/multi-juicer/main/wrongsecrets-balancer/ui/public/favicon.ico
+  #      showVersionNumber: false
+  #      showGitHubLinks: false
+  #    challenges:
+  #      showHints: true
+  #    hackingInstructor:
+  #      isEnabled: true
+  #    ctf:
+  #      showFlagsInNotifications: false
```

Comment:
Can you check which of these are functional and which ones can go please?

This is also the stuff that we might want to replace with functional settings for WrongSecrets

Reply:
As discussed with @bendehaan, should be removed. 
Done now